honeybee_3dm/helper.py

```diff
@@ -26,3 +35,187 @@ def get_unit_system(file_3dm):
+    from the config file.
+
+    Args:
+        config: A dictionary of the config settings.
+        face_obj: A Ladybug Face3d object.
+        name: A text string of the name of the rhino object.
+        layer_name: A text string of the rhino layer name.
+
+    Returns:
+        A Honeybee Face object.
+    """
+    
+    obj_name = name or clean_and_id_string(layer_name)
+    args = [clean_string(obj_name), face_obj]
+    
+    if config['layers'][layer_name]['honeybee_face_type'] == 'roof':
+        face_type = face_types.roof_ceiling
+    
+    elif config['layers'][layer_name]['honeybee_face_type'] == 'wall':
+        face_type = face_types.wall
+    
+    elif config['layers'][layer_name]['honeybee_face_type'] == 'floor':
+        face_type = face_types.floor
+    
+    elif config['layers'][layer_name]['honeybee_face_type'] == 'airwall':
+        face_type = face_types.air_boundary
+    
+    args.append(face_type)
+    hb_face = Face(*args)
+    hb_face.display_name = args[0]
+    
+    if 'radiance_material' in config['layers'][layer_name]:
+        radiance_modifiers = mat_to_dict(config['sources']['radiance_material'])
+        hb_face.properties.radiance.modifier = radiance_modifiers[config
+            ['layers'][layer_name]['radiance_material']]
+        return hb_face
+    else:
+        return hb_face
+
+
+def face3d_to_hb_face_with_rad(config, face_obj, name, layer_name):
+    """Create a Honeybee Face object with a radiance material assigned to it.
+
+    Args:
+        config: A dictionary of the config settings.
+        face_obj: A Ladybug Face3d object.
+        name: A text string of the name of the rhino object.
+        layer_name: A text string of the rhino layer name.
+
+    Returns:
+        A Honeybee Face object.
+    """
+    
+    obj_name = name or clean_and_id_string(layer_name)
+    args = [clean_string(obj_name), face_obj]
+    hb_face = Face(*args)
+    hb_face.display_name = args[0]
+    
+    if 'radiance_material' in config['layers'][layer_name]:
+        radiance_modifiers = mat_to_dict(config['sources']['radiance_material'])
+        hb_face.properties.radiance.modifier = radiance_modifiers[config
+            ['layers'][layer_name]['radiance_material']]
+        return hb_face
+    else:
+        return hb_face
+
+
+def face3d_to_hb_object(config, face_obj, name, layer_name):
+    """Create Honeybee Aperture, Shade, and Door objects.
+
+    Args:
+        config: A dictionary of the config settings.
+        face_obj: A Ladybug Face3d object.
+        name: A text string of the name of the rhino object.
+        layer_name: A text string of the rhino layer name.
+
+    Returns:
+        A tuple of lists. Each list contains Honeybee Aperture objects, 
+            Honeybee Shade objects, and Honeybee Door objects. List will
+            be empty if no objects are found for that Honeybee object.
+    """
+
+    hb_shades, hb_apertures, hb_doors = tuple([[] for _ in range(3)])
```

Comment:
What about?

```suggestion
    hb_shades, hb_apertures, hb_doors = ([], [], [])
```

If you still want to use a loop you should remove the extra []:

```python
hb_shades, hb_apertures, hb_doors = tuple([] for _ in range(3))
```